tests/test_crd.rs

```diff
@@ -65,3 +65,99 @@ async fn k8s_test_test_ensure_crd_created() {
         .await
         .expect("CRD should be created"))
 }
+
+#[tokio::test]
+#[ignore = "Tests depending on Kubernetes are not ran by default"]
+async fn k8s_test_wait_for_crds() {
+    // TODO: Switch this to using TemporaryResource from the integration-test-commons crate
```

Comment:
Every TODO should have a ticket (I'm not sure when the best time is for adding those references either)